IdentityCore/src/oauth2/MSIDTokenResponse.h

```diff
@@ -44,7 +44,10 @@
  */
 @property (nonatomic) NSInteger expiresOn;
 @property (nonatomic, nullable) NSString *accessToken;
+// In AT Pop scenario, we need to pass back the tokenType and kid(req_conf)
```

Comment:
Looks like we need to both pass back from response and pass in to the request? If so please update the comment. Correct me if I misunderstood :)

Reply:
The kid value is generated in calling app (MSAL) then pass to the broker request as a parameter. Then broker will request an AT bound to that kid and return both the AT and kid back to MSAL. I think this class is for creating/parsing the response from broker -> MSAL, It already works in legacy broker flow and this update will fix the SSO Ext broker flow.